universal-application-tool-0.0.1/app/views/admin/questions/QuestionEditView.java

```diff
@@ -66,6 +85,8 @@ private ContainerTag buildEditQuestionForm(QuestionDefinition definition) {
   private ContainerTag buildQuestionForm(QuestionForm questionForm) {
     ContainerTag formTag = form().withMethod("POST");
     formTag
+        .with(hiddenInputWithValue("questionId", questionForm.getQuestionIdString()))
```

Comment:
Question (blocking): the question ID is available via the URL path and is passed to the `update` method in the controller, it shouldn't need to also be included in the form contents

Reply:
Done.